exercises/practice/sum-of-multiples/.meta/example.jl

```diff
@@ -0,0 +1,11 @@
+function sum_of_multiples(limit, multiples)
+    uniques = Set([0]) # avoid empty set, which would make sum() unhappy
```

Comment:
> It might be better to use a BitSet anyway.

It's not an examplar solution in this case, so as long as it works, it's good enough. We can merge it and optimise later if we want.